src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -632,8 +632,8 @@ bool ReconstructionEngine_sequentialSfM::bundleAdjustment(std::set<IndexT>& newR
     const std::size_t nbRefinedPoses = _localStrategyGraph->getNbPosesPerState(BundleAdjustment::EParameterState::REFINED);
     const std::size_t nbConstantPoses = _localStrategyGraph->getNbPosesPerState(BundleAdjustment::EParameterState::CONSTANT);
 
-    // restore the Dense linear solver type if the number of cameras in the solver is <= 100
-    if(nbRefinedPoses + nbConstantPoses <= 100)
```

Comment:
wouldn't be better to have it as a member of the class so it is a parameter that can be eventually set?

Reply:
Yes, we can expose it.
In that case, we should also expose the threshold for the first N cameras too:
https://github.com/alicevision/AliceVision/pull/583/files#diff-cc29faeac378cc9e42bfb073bd95ee9cL608

I will do it in another PR.